examples/intro_data_profiler.ipynb

```diff
@@ -0,0 +1,442 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "60af6856",
+   "metadata": {},
+   "source": [
+    "# DataProfiler - What's in your data?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3297611c",
+   "metadata": {},
+   "source": [
+    "This introductory jupyter notebook demonstrates the basic usages of the DataProfiler. The library is designed in the way that users can easily get the statistics and other detailed information about the input datasets with just several lines of code. DataProfiler provides multiple data classes that can handle different types of data. In addition, users are given various options to skip some properties if not needed while profiling their datasets. The last key feature covered in this example is the ability to allow users to update their profilers from multiple batches of the large datasets, or merge multiple profilers which is suitable for the distributed computing environment. In particular, this example covers the followings:\n",
+    "\n",
```

Comment:
I think I would say the different types of data it can handle. I think "users are given various options to skip some properties if not needed while profiling their datasets" is too vague for the average user they wont know what that means. 
"update their profilers" -> "update their profiles"
"merge multiple profilers which is suitable for the distributed computing environment." -> "merge multiple profiles."